setup.py

```diff
@@ -75,12 +75,9 @@
     "pydot", # For graphs
     "python-dateutil",
     "colorama",
+    "llvmlite",
 ]
 
-if not ((not is_PyPy and sys.version_info >= (3, 8)) or (is_PyPy and sys.version_info >= (3, 6))):
-    INSTALL_REQUIRES.append("llvmlite")
```

Comment:
Removing this is the part I don't agree with. 

Probably should go the other way and *always*  require "llvmlite". See the other comment for more detail here.

Reply:
But we are _always_ requiring... See L78, I've added `"llvmlite"` to `INSTALL_REQUIRES` again.

Reply:
Ah - Ok. Then this is just about simplifying the code - great! I don't have any suggestions then. Doing via conda sound fine.